pilot/pkg/security/authz/builder/testdata/http/custom-multiple-http-provider-1-out1.yaml

```diff
@@ -0,0 +1,47 @@
+name: envoy.filters.http.rbac
+typedConfig:
+  '@type': type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC
+  shadowRules:
+    action: DENY
+    policies:
+      istio-ext-authz-ns[foo]-policy[httpbin-1]-rule[0]:
```

Comment:
Dette tror jeg er det de kaller "shadow-rule".
**Fra designdok'en referet til i kildekoden:**
_A request is allowed if and only if "CUSTOM not rejected" AND "DENY not matched" AND "ALLOW matched".

The CUSTOM action is only triggered when the matching rule evaluates to true.

* Istio will inject an extra RBAC filter in shadow mode to evaluate the matching rules before the ext_authz filter, and store the matching result to a well-known dynamic metadata
  namespace. The shadow mode means the RBAC filter will only evaluate but not enforce. 

* Later in the ext_authz filter, it will read from the dynamic metadata via the metadata matcher to decide whether or not to trigger the external flow based on the matching result._

Det jeg ikke skjønner er hvorfor info fra begge providerene kommer med i begge shadow-rule'ene, ville trodde at denne delen (istio-ext-authz-ns[foo]-policy[httpbin-1]-rule[0]:) skulle med for provider-1 og 
den andre (istio-ext-authz-ns[foo]-policy[httpbin-2]-rule[0]) skulle med for provider-2 ??